test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -184,10 +189,13 @@ class StructuredFormDataBuilderSpec extends Spec {
         "2_field" -> "value1,value3, value4"
       ),
       objectStructure(
-        "field" -> arrayNode(
-          arrayNode(textNode("value1"), textNode("value2")),
-          arrayNode(textNode("value1"), textNode("value3"), textNode("value4"))
-        )
+        field(
+          "field",
+          arrayNode(
+            arrayNode(textNode("value1"), textNode("value2")),
+            arrayNode(textNode("value1"), textNode("value3"), textNode("value4"))
+          ),
+          Map.empty)
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.